### PR TITLE
Create CONTRIBUTING.md #914

### PR DESCRIPTION
**Pull request by ksheremet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# How to contribute to Delern Flashcards
+
+I'm really glad you're reading this, because we need volunteer developers to help this project to become great!🎉👍
+
+## Contributing code in Mobile App
+
+🚀Install environment
```
Comment (dotdoom) [inline CONTRIBUTING.md]: readability: use `:smile:` format for emoji rather than unicode characters.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+1. Install Flutter SDK and IDE from [here](https://flutter.io/docs/get-started/install/)
+2. Fork `https://github.com/dasfoo/delern` into your own GitHub account. If you already have a fork, and are now installing a development environment on a new machine, make sure you've updated your fork so that you don't use stale configuration options from long ago.
```
Comment (dotdoom) [inline CONTRIBUTING.md]: readability: avoid lines longer than 80 characters.

Comment (dotdoom) [inline CONTRIBUTING.md]: nit: only use `1. ` for numbered lists, and let Markdown take care of numeration.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+3. If you haven't configured your machine with an SSH key that's known to github, then follow [GitHub's directions](https://help.github.com/articles/generating-ssh-keys/) to generate an SSH key.
+4. `git clone git@github.com:<your_name_here>/delern.git`
+5. `cd delern`
+6. `git remote add upstream git@github.com:dasfoo/delern.git` (So that you fetch from the master repository, not your clone, when running `git fetch`
+   et al.)
```
Comment (dotdoom) [inline CONTRIBUTING.md]: unnecessary newline?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+4. Run `flutter analyze` in `delern/flutter` directory to check whether code complies with style guide
+5. Fix style guide warnings
+6. `git commit -a -m "<your informative commit message>"`
+7. `git push origin <name_of_your_branch>`
+
+Please make sure all your checkins have detailed commit messages explaining the patch.
+
+🎉To send us a pull request:
+1. `git pull-request` (if you are using [Hub](http://github.com/github/hub/)) or go to `https://github.com/dasfoo/delern` and click the "Compare & pull request" button
+
```
Comment (dotdoom) [inline CONTRIBUTING.md]: unnecessary newlines?

File: CONTRIBUTING.md (hunk 2 above)
Comment (dotdoom) [inline CONTRIBUTING.md]: Should this be a clickable link as well?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ksheremet) [reply CONTRIBUTING.md]: I'll do changes but I think it is not necessary to spend time on emoji

**Pull request merged**